### PR TITLE
2684 workflows

### PR DESCRIPTION
**Pull request by larisa17**
This fixes https://github.com/gitcoinco/passport/issues/2684

### CODE REVIEW

File: .github/actions/test/action.yml
Comment (nutrina) [inline .github/actions/test/action.yml:1]: If this is not used, it can be deleted.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -44,20 +47,3 @@ jobs:
         run: yarn test:ceramic-integration
         env:
           CERAMIC_CLIENT_URL: http://localhost:7007
-
```
Comment (nutrina) [inline .github/workflows/ci.yml:47]: I would expect this to not be removed.
If we have a separate action for this, invoke it here

Comment (larisa17) [reply .github/workflows/ci.yml:47]: So we want to check the provider bitmaps at every PR ?

**Review by nutrina — CHANGES_REQUESTED** [review]
Please see my comments

File: .github/workflows/ci.yml (hunk 1 above)
Comment (nutrina) [reply .github/workflows/ci.yml:47]: Yes.

**Pull request merged**